CMakeLists.txt

```diff
@@ -133,12 +133,20 @@ include(GNUInstallDirs)
 include(CMakePackageConfigHelpers)
 
 # Library files
-install(TARGETS shout
+install(TARGETS shout icecast-httpp icecast-net icecast-timing icecast-avl
```

Comment:
Why do you want to export the targets from the other project?

Reply:
CMake errors without this.

Reply:
Why? Can you post the error message. This doesn't look right.

Reply:
I don't have the error message easily accessible anymore, but I remember CMake complained that targets that shout depends on weren't exported.